test/features/google-login/google-login.test.js

```diff
@@ -0,0 +1,84 @@
+import sinon from 'sinon';
+import { expect } from '@esm-bundle/chai';
+import { readFile } from '@web/test-runner-commands';
+import initGoogleLogin from '../../../libs/features/google-login.js';
+
+describe('Google Login', () => {
+  let initializeSpy;
+  let promptSpy;
+  beforeEach(async () => {
+    document.body.innerHTML = await readFile({ path: './mocks/google-login.html' });
+    window.google = window.google || {
+      accounts: {
+        id: {
+          initialize: () => {},
+          prompt: () => {},
+        },
+      },
+    };
+    initializeSpy = sinon.spy(window.google.accounts.id, 'initialize');
+    promptSpy = sinon.spy(window.google.accounts.id, 'prompt');
+    window.adobeid = {
+      client_id: 'milo',
+      scope: 'gnav',
+    };
+  });
+
+  afterEach(() => {
+    document.body.innerHTML = '';
+    initializeSpy.restore();
+    promptSpy.restore();
```

Comment:
Should we also delete `window.adobeid` to not pollute other tests?

Reply:
I did not find it necessary, but I can add it for peace of mind.